src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Application {
+    public String getRandomBall() {
```

Comment:
```suggestion
    public Ball getRandomBall() {
```
this method should return a ball

Reply:
I do not quite understand your suggestion :( Should I change in the method String to Ball and then return Ball? (String was there by default). Or should I keep String and return ball.toString(); instead of writing the message in return statement in Lottery class?

Reply:
You should return Ball)